README.md

```diff
@@ -7,23 +7,23 @@
 [![Build Status](https://travis-ci.org/vscode-icons/vscode-icons.svg?branch=master)](https://travis-ci.org/vscode-icons/vscode-icons)
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/vscode-icons/vscode-icons?branch=master&svg=true)](https://ci.appveyor.com/project/robertohuertasm/vscode-icons)
 
-[![Dependencies Status](https://david-dm.org/vscode-icons/vscode-icons/status.svg?=1)](https://david-dm.org/vscode-icons/vscode-icons)
-[![DevDependencies Status](https://david-dm.org/vscode-icons/vscode-icons/dev-status.svg?=1)](https://david-dm.org/vscode-icons/vscode-icons?type=dev)
-[![Greenkeeper badge](https://badges.greenkeeper.io/vscode-icons/vscode-icons.svg?=1)](https://greenkeeper.io/)
+[![Dependencies Status](https://david-dm.org/vscode-icons/vscode-icons/status.svg)](https://david-dm.org/vscode-icons/vscode-icons)
+[![DevDependencies Status](https://david-dm.org/vscode-icons/vscode-icons/dev-status.svg)](https://david-dm.org/vscode-icons/vscode-icons?type=dev)
+[![Greenkeeper badge](https://badges.greenkeeper.io/vscode-icons/vscode-icons.svg)](https://greenkeeper.io/)
 
-[![codecov](https://codecov.io/gh/vscode-icons/vscode-icons/branch/master/graph/badge.svg?=1)](https://codecov.io/gh/vscode-icons/vscode-icons)
-[![Known Vulnerabilities](https://snyk.io/test/github/vscode-icons/vscode-icons/badge.svg?=1)](https://snyk.io/test/github/vscode-icons/vscode-icons)
+[![codecov](https://codecov.io/gh/vscode-icons/vscode-icons/branch/master/graph/badge.png)](https://codecov.io/gh/vscode-icons/vscode-icons)
```

Comment:
Resolved with https://github.com/Microsoft/vscode-vsce/pull/185. Update `vsce` to `1.28.0+`.